src/main/java/com/softserve/actent/service/impl/ChatServiceImpl.java

```diff
@@ -1,19 +1,28 @@
 package com.softserve.actent.service.impl;
 
+import com.softserve.actent.constant.ExceptionMessages;
+import com.softserve.actent.exceptions.ResourceNotFoundException;
+import com.softserve.actent.exceptions.codes.ExceptionCode;
+import com.softserve.actent.exceptions.validation.IncorrectChatTypeException;
 import com.softserve.actent.model.entity.Chat;
 import com.softserve.actent.model.entity.ChatType;
 import com.softserve.actent.repository.ChatRepository;
 import com.softserve.actent.service.ChatService;
+import lombok.extern.slf4j.Slf4j;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.stereotype.Service;
 import javax.transaction.Transactional;
-import java.util.Optional;
 
+@Slf4j
```

Comment:
We use @Log4j2

Reply:
yeah
i rewrite it
you can check and aprove changes